tests/acceptance/features/bootstrap/AuthContext.php

```diff
@@ -212,11 +212,15 @@ public function userRequestsEndpointsWithNoAuthentication($method, TableNode $ta
 			$this->featureContext->pushToLastHttpStatusCodesArray(
 				$this->featureContext->getResponse()->getStatusCode()
 			);
-			$this->featureContext->pushToLastOcsCodesArray(
-				$this->featureContext->ocsContext->getOCSResponseStatusCode(
-					$this->featureContext->getResponse()
-				)
-			);
+			try {
+				$this->featureContext->pushToLastOcsCodesArray(
+					$this->featureContext->ocsContext->getOCSResponseStatusCode(
+						$this->featureContext->getResponse()
+					)
+				);
+			} catch (\Exception $e) {
+				// if response couldn't be converted into xml then skip push to last ocs status codes array
```

Comment:
Maybe we should push a "special" OCS status code here. For example:
```suggestion
				// if response couldn't be converted into xml then push "notset" to last ocs status codes array
				$this->featureContext->pushToLastOcsCodesArray('notset');
```

Otherwise the test might pass if suddenly we stop getting any OCS status codes returned, and it will not be obvious what happened.

And then we can make a scenario for OCIS that expects something like:
```
Then the OCS status code of responses on all endpoints should be "notset"
```

And make an issue in OCIS about the the HTTP 401 responses that do not also have OCS 997.

Reply:
There is already an issue reporting such behavior, I have tagged affected scenarios with the issue tag!